libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -299,6 +310,15 @@ float AP_BattMonitor::current_total_mah(uint8_t instance) const {
     }
 }
 
+/// consumed_wh - returns energy consumed since start-up in watt-hours
+float AP_BattMonitor::consumed_wh(uint8_t instance) const {
+    if (instance < _num_instances) {
+        return _BattMonitor_STATE(instance).consumed_wh;
```

Comment:
`_BattMonitor_STATE()` macro no longer exists in master. `state[instance]` is the appropriate substitution.